### DISCUSSION TITLE
Terminal foreground color doesn't change. Anyone know what I'm doing wrong?

### DISCUSSION THREAD

**Discussion by guppy57** [zed-industries/zed#11418] [Feature Requests]
Hey y'all, trying to make the switch to Zed from VSCode. For the most part I've been able to tweak my theme exactly how I want it. However, i've not been able to get the `terminal.foreground` to actually change.

I use zsh and oh-my-zsh with the agnoster theme.

![WebP to PNG foreground color issue](https://github.com/zed-industries/zed/assets/33143849/69356f3c-ac13-4dce-8c73-e97f3cdcae19)

![ (1)](https://github.com/zed-industries/zed/assets/33143849/48c25128-96b3-4eb8-9fe5-831a847acfeb)

**Comment by zakj** [comment]
I _think_ the fix is trivial: https://github.com/zed-industries/zed/pull/14281

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
